crates/criticalup-cli/src/errors.rs

```diff
@@ -162,6 +162,13 @@ pub(crate) enum Error {
     )]
     ManifestAlreadyExists(),
 
+    #[error("\
+        An installation corresponding to your `criticalup.toml` was not found.\n\
+        Please ensure your `criticalup.toml` is present and what you expect, then\n\
+        run `criticalup install` to ensure it is installed.
```

Comment:
Please ignore if you think otherwise but my basic thinking behind these changes/suggestions:

1. This error will only show if criticalup finds a criticalup.toml either in the current directory or via `--project /path/to/criticalup.toml` flag. In either case, criticalup.toml is present. So, we need to just ask about their expectations.
2. Some englishes. xD

```suggestion
        An installation corresponding to your `criticalup.toml` was not found.\n\
        Please ensure your `criticalup.toml` contains what you expect, and then\n\
        run `criticalup install` to complete the installation.
```